### PR TITLE
Add the doc about the endpoint that allows to upsert a list of attrib…

### PR DESCRIPTION
**Pull request by NolwennP**
…ute options

### CODE REVIEW

File: content/swagger/resources/attributes/routes/attributes_code_options.yaml [hunk 1]
```diff
@@ -125,3 +125,47 @@ post:
...
+  x-body-by-line: Contains several lines, each line is an attribute option in JSON standard format
+  parameters:
+    - $ref: '#/parameters/attribute_code'
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/AttributeOption"
+        x-examples:
+          x-example-1:
+            '{"code":"black", "labels":{"en_US": "Black","fr_FR": "Noir"}}'
```
Comment (ahocquard) [inline content/swagger/resources/attributes/routes/attributes_code_options.yaml]: add `"attribute": "attribute_code"`in one line please, to show that it is optional
and a second line with not the same `attribute_code` to show that it creates an error

File: content/swagger/resources/attributes/definitions/attribute_option.yaml [hunk 2]
```diff
@@ -1,14 +1,17 @@
 type: object
+required: ["code"]
 properties:
   code:
     type: string
     description: Code of option
   attribute:
     type: string
     description: Code of attribute related to the attribute option
+    x-validation-rules: It is the same attribute code that the one given as path parameter
```
Comment (anaelChardan) [inline content/swagger/resources/attributes/definitions/attribute_option.yaml]: it is the same attribute code `than`

Comment (ahocquard) [reply content/swagger/resources/attributes/definitions/attribute_option.yaml]: no, `as`

https://forum.wordreference.com/threads/same-than-same-as-same-that.400069/

Comment (anaelChardan) [reply content/swagger/resources/attributes/definitions/attribute_option.yaml]: I was not talking about the `as` but the part `It is the same attribute code that` I don't know if the `that` is correct. I check on several websites ;).

Comment (ahocquard) [reply content/swagger/resources/attributes/definitions/attribute_option.yaml]: You didn't check my link. 
`It is the same attribute code as the one given `

**Pull request merged**